ui/src/index.scss

```diff
@@ -248,4 +248,74 @@ $tracking in $fontTrackings {
   }
 }
 
-a:link, a:visited, a:hover, a:active { color: var(--wave-themePrimary); }
\ No newline at end of file
+a:link, a:visited, a:hover, a:active { color: var(--wave-themePrimary); }
+
+/* Default markdown syntax highlighting theme - androidstudio. */
+pre code.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 1em;
+}
+
+code.hljs {
+  padding: 3px 5px;
+}
```

Comment:
Why was hljs css moved into the global one?

Reply:
> Because I have no control over hljs- prefixed internal classes from highlight.js so I couldn't replace it with css modules.

Not sure what you mean.

> And these classes were in global one before https://github.com/h2oai/wave/pull/2189 anyway.

Correct, doesn't matter since they end up in the same CSS file in prod anyway.